crypto/bio/bio_addr.c

```diff
@@ -238,7 +238,7 @@ static int addr_strings(const BIO_ADDR *ap, int numeric,
             } else
 # endif
             {
-                ERR_raise_data(ERR_LIB_BIO, ERR_R_SYS_LIB, gai_strerror(ret));
+                ERR_raise_data(ERR_LIB_BIO, ERR_R_SYS_LIB, gai_strerrorW(ret));
```

Comment:
I expect this to fail on `VC-WIN64A`, because `ERR_raise_data()` takes a `const char *`, not a `const wchar_t *`.

Reply:
See https://github.com/openssl/openssl/runs/5957890721?check_suite_focus=true.